src/tribler/core/components/gigachannel_manager/tests/test_gigachannel_manager.py

```diff
@@ -113,32 +111,40 @@ def test_updated_my_channel(personal_channel, gigachannel_manager, tmpdir):
     gigachannel_manager.download_manager.start_download.assert_called_once()
 
 
-async def test_check_and_regen_personal_channel_torrent(personal_channel, gigachannel_manager):
+async def test_check_and_regen_personal_channel_torrent_wait(personal_channel, gigachannel_manager):
     with db_session:
         chan_pk, chan_id = personal_channel.public_key, personal_channel.id_
```

Comment:
Actually, db_session is not necessary here, as the `personal_channel` object is already linked to some (finished) db_session:
```suggestion
    chan_pk, chan_id = personal_channel.public_key, personal_channel.id_
```

Reply:
Fixed